scripts/installer/Dockerfile

```diff
@@ -1,8 +1,7 @@
-FROM quay.io/centos/centos:stream8
+FROM quay.io/centos/centos:stream9
 
-RUN dnf -y install https://yum.puppetlabs.com/puppet7/puppet7-release-el-8.noarch.rpm
+RUN dnf -y install https://yum.puppetlabs.com/puppet7/el/9/x86_64/puppet7-release-7.0.0-19.el9.noarch.rpm
```

Comment:
I couldn't find a `puppet7-release-el-9.noarch.rpm` so I just used this one. Probably not best to hard code a specific version like this.

Reply:
```suggestion
RUN dnf -y install https://yum.puppet.com/puppet7-release-el-9.noarch.rpm
```